docs/search-best-practice.md

```diff
@@ -0,0 +1,68 @@
+# Best Practices for Documentation Search
+Documentation (docs) search is an important part of developer experience. It makes life so easy for developers by enabling them narrow down to sections they’re looking for in the documentation. Choosing the right tool for docs search can be a hassle. 
+
+Here at CNCF,we’ve explored different search tools and we’ve decided to narrow our options to this 3:
+
+1. Lunr
+1. Google search
+1. Docsearch by Algolia
+
+# Pros And Cons
```

Comment:
I suggest removing this title, and bumping the others up one level on the hierarchy.